### PR TITLE
Wrap embedded statements over multiple lines.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
This is the result of running the latest analyzer in roslyn-analyzers against Roslyn.sln. This will be needed to move to the latest version for the team.

### CODE REVIEW

File: eng/targets/Imports.targets [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
 
+  <!--
+    Work around .editorconfig evaluation bugs in command line builds.
+    https://github.com/dotnet/roslyn/issues/43371
+  -->
+  <ItemGroup Condition="'$(DesignTimeBuild)' != 'true'">
+    <_AbsoluteCompile Include="@(Compile->'%(FullPath)')" />
+    <Compile Remove="@(Compile)" />
+    <Compile Include="@(_AbsoluteCompile)" />
+  </ItemGroup>
```
Comment (sharwell) [inline eng/targets/Imports.targets:15]: @dotnet/roslyn-infrastructure please review this workaround for #43371

Comment (mavasani) [reply eng/targets/Imports.targets:15]: Looks good to me.

Comment (jaredpar) [reply eng/targets/Imports.targets:15]: @agocke  and @chsienki need to look at this. At a glance it looks okay.

Comment (agocke) [reply eng/targets/Imports.targets:15]: LGTM, although this indicates to me that there's no compiler bug here, but some sort of MSBuild targets problem.

Comment (chsienki) [reply eng/targets/Imports.targets:15]: The actual change seems fine, I'm curious to understand the underlying bug though. Can we not  just look at the full name directly inside the compiler?

**Pull request merged**